docs/README.configuration.md

```diff
@@ -90,9 +93,22 @@ updated on receiving ICMP packet too big messages.
 policy_icmpv4_incoming = policies['ALLOW'],
 policy_icmpv6_incoming = policies['ALLOW'],
 policy_icmpv4_outgoing = policies['ALLOW'],
-policy_icmpv6_outgoing = policies['ALLOW']
+policy_icmpv6_outgoing = policies['ALLOW'],
 ```
 
 Snabb-lwaftr can be configured to ALLOW or DROP incoming and outgoing ICMPv4
 and ICMPv6 messages. If a finer granularity of control is desired, contact the
 development team via github or email.
+
+```lua
+v4_vlan_tag = C.htonl(0x81000444),
```

Comment:
Like the previous one, this is not a blocker for merging, just some food for thought:

This is a bit crude for a configuration file. Also, the whole `ffi.C` namespace is exported to the scope of the configuration file, which opens a can of worms, as one could as well do something like this from the configuration:
```lua
setting = C.abort_()  -- Ooops!
```
We could just expose a fixed set of functions to the configuration file scope, using `setfenv()` and `pcall()` to limit the harm that can be done from config files, and actually the return values from `pcall()` could be used to report the location in the file where there are syntax errors and so on.

As said, this is not a blocker: At the current stage things are pretty in flux, and whoever touches the configuration is expected to know what's going on and not shoot themselves in the foot, so it is better to have the extra flexibility for now.